filebeat/beater/filebeat.go

```diff
@@ -119,16 +119,21 @@ func (fb *Filebeat) Run(b *beat.Beat) error {
 	if err != nil {
 		return err
 	}
-	// Stop crawler -> stop prospectors -> stop harvesters
-	defer crawler.Stop()
-
 	// Blocks progressing. As soon as channel is closed, all defer statements come into play
+
+	// Stop crawler -> stop prospectors -> stop harvesters
 	<-fb.done
+	crawler.Stop()
```

Comment:
Can you please add a note here why we don't use defer for the crawler.Stop? I already see me adding a defer to it in 3 months :-)